web/views/command_version_view.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Cog.V1.CommandVersionView do
   use Cog.Web, :view
 
+  alias Cog.Commands.Help.CommandFormatter
```

Comment:
Not sure exactly where it needs to ultimately reside, but it feels even weirder for that view to be reaching into the Cog command-specific code hierarchy for its implementation.